wiki/Disambiguation/Rank/id.md

```diff
@@ -2,14 +2,12 @@
 tags:
   - grade
   - ranking
-outdated: true
-outdated_since: b7baabcd29d52faed43f86aa42912bd1cb50ebed
 ---
 
 # Rank
 
 Rank mempunyai tiga arti:
 
-- Sama seperti [huruf pada grade](/wiki/Grade).
-- Posisi seorang pemain pada papan peringkat online (umumnya dikenal sebagai [ranking pemain](/wiki/Ranking)).
+- Sebuah [grade](/wiki/Grade) berbentuk huruf.
+- [Ranking pemain](/wiki/Ranking) atau posisi seorang pemain pada papan peringkat online.
 - Kata kerja dari "menuju ranking", yang artinya sedang melewati [prosedur beatmap ranking](/wiki/Beatmap_ranking_procedure).
```

Comment:
Maybe `menuju ranking` can be changed to `menuju ranked` or `menjadi ranked`.

Reply:
good idea